apps/api/src/app/workflows-v2/workflow.controller.e2e.ts

```diff
@@ -787,6 +787,51 @@ describe('Workflow Controller E2E API Testing #novu-v2', () => {
       expect(res.error!.message).to.contain('Workflow');
       expect(res.error!.ctx?.workflowId).to.contain('123');
     });
+
+    it('should create unique step IDs when duplicating workflow with multiple steps of same type', async () => {
```

Comment:
@cursoragent test is failing: 1) Workflow Controller E2E API Testing #novu-v2
       Duplicate workflow
         should create unique step IDs when duplicating workflow with multiple steps of same type:

      Step 0 should have different ID after duplication
      + expected - actual